src/network/modules/tokens/tokens.ts

```diff
@@ -117,6 +133,13 @@ function makeTokensRemoveExtendedData(
   return data
 }
 
+function makeTokensMintBurnParam(param: TokensMintBurnParam) {
+  const data = new Map()
+  data.set(0, param.symbol)
+  data.set(1, new Map(Object.entries(param.addresses)))
+  return data
+}
```

Comment:
Thank you @hansl for making the `@param` for these two methods the same. Much appreciated!